### DISCUSSION TITLE
[BUG] Send-PnPMail fails to send mail

### DISCUSSION THREAD

**Discussion by HeKr09** [pnp/powershell#3235] [General] [answered]
### Reporting an Issue or Missing Feature
I want to send a mail with Send-PnPMail to existing mail addresses but it fails for always the same mails although other tools (e.g. Power Automate) can send mails.

### Expected behavior 
I expect that Send-PnPMail sends a mail.

### Actual behavior 
Currently I am receiving the following error:
```
Send-PnPMail: D:\PATH\Check-DataInconsistency.ps1:87
Line |
  87 |              Send-PnPMail -To $currentMailbox -Subject "$customer // I …
     |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The e-mail message cannot be sent. Make sure the e-mail has a valid recipient.
```
The same scripts works with other mailboxes but some are failing.

### Steps to reproduce behavior
I am running the following send command:
`Send-PnPMail -To $currentMailbox -Subject "$customer // Inconsistency detected" -Body "There is an inconsistency at items with IDs: $($correctableIds -join ", "). Please go to the list of the customer, add the field 'ID' to the current view, filter the view by the IDs specified and check the entries."`

`$currentMailbox` is filled with a valid mail address. I checked spelling and everything and send a test mail - worked.
**If I use another mailbox everything works fine.
If I use the -From parameter to send it from a mailbox within the same tenant it also works.
If I use the -CC parameter the mail is sent but only to the CC recipients without an erro.**

### What is the version of the Cmdlet module you are running?
2.1.1

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
hi @HeKr09 , looks like something specific to that mailbox.

If it works for one mailbox and not the other, it is not an issue with PnP PowerShell or the cmdlet itself.  
Could be something related to that mailbox or some other settings.
Am moving this to discussion.
Would suggest checking the difference between those mailbox , the one which works and one which doesn't.

**Reply by HeKr09** [reply]
I already checked differences. There are none that I could found.

**Comment by HeKr09** [comment]
Can anybody help me?

**Reply by gautamdsheth** [reply]
Could it be permission issue ? Maybe try adding the user/mailbox to the SharePoint site ?

https://powerusers.microsoft.com/t5/Using-Flows/The-e-mail-message-cannot-be-sent-Make-sure-the-e-mail-has-a/td-p/1218477

**Comment by HeKr09** [comment, marked as answer]
I guess the problem was that the user never has been signed in on the site. After logging in it worked.
